src/app/organizations/collection/add-entry/add-entry.component.html

```diff
@@ -0,0 +1,50 @@
+<h1 mat-dialog-title>Add to collection</h1>
+<div mat-dialog-content>
+  <app-alert></app-alert>
+
+  <ng-template #loading>
```

Comment:
app-alert has the progress bar already. this will likely have two progress bars at some point